app/services/send_apply_again_call_to_action.rb

```diff
@@ -0,0 +1,22 @@
+# This worker will be as a one-off task to inform previously unsuccessful candidates about apply again
+class SendApplyAgainCallToAction
+  include Sidekiq::Worker
+
+  def perform
+    unsuccessful_application_forms.find_each do |application_form|
+      CandidateMailer.apply_again_call_to_action(application_form).deliver
+    end
+  end
+
+private
+
+  def unsuccessful_application_forms
+    ApplicationForm
```

Comment:
I wonder if it's worth explicitly excluding applications that have already started apply_again, in case there's a significant delay between launching the feature and sending the email?